baselines/baseline_template/pyproject.toml

```diff
@@ -44,7 +44,7 @@ hydra-core = "1.3.2" # don't change this
 [tool.poetry.dev-dependencies]
 isort = "==5.11.5"
 black = "==23.1.0"
-docformatter = "==1.5.1"
+docformatter = "==1.7.1"
```

Comment:
Is this intentional?

Reply:
The latest version of docformatter is `1.7.5`

Reply:
No, it wasn't. I'll remove that.